crates/pgrepr/src/error.rs

```diff
@@ -32,6 +32,9 @@ pub enum PgReprError {
 
     #[error("Internal error: {0}")]
     InternalError(String),
+
+    #[error("{0}")]
+    String(String),
```

Comment:
I dislike strongly typed errors that end up just being strings: but I don't have a better suggestion

Reply:
I agree, but I've also found creating specific variants to cause friction when developing. That combined with reusing certain variants sometimes makes debugging trickier than needed. String isn't elegant, but it's practical for constructing debuggable errors.